runtime/src/main/kotlin/kotlin/random/Random.kt

```diff
@@ -7,7 +7,7 @@ package kotlin.random
 import kotlin.native.concurrent.AtomicLong
 import kotlin.system.getTimeNanos
 
-public abstract class NativeRandom {
+internal class NativeRandom {
```

Comment:
Should we keep default implementation as companion object or we can make it just a top-level object?

Reply:
I think we can do make it a top level